website/views.py

```diff
@@ -3,7 +3,8 @@
 from django.http import JsonResponse, HttpResponse
 from django.views import View
 
-from tg_routine.commandHelpers import *
+from tg_routine.commandHelpers import write_specific_helper
```

Comment:
from tg_routine.commandHelpers import *

Means that everything will be imported from the commandHelpers, no need to change that